.tools/test/SPECIFICATION.md

```diff
@@ -0,0 +1,44 @@
+# Deployment Instructions
+These instructions assume a "target account" (where tests will be invoked) of "python" per [this repository's configuration](config/targets.yaml).
+You can replace Python with any of the other languages listed in this repository's configuration.
+
+To request an alternate configuration for your own repository, please [submit an issue](https://github.com/awsdocs/aws-doc-sdk-examples/issues/new?assignees=octocat&labels=type%2Fenhancement&labels=Tools&title=%5BEnhancement%5D%3A+Weathertop+Customization+Request&&) with the `Tools` label.
```

Comment:
Shouldn't the `assignees` be `ford-at-aws`?

Reply:
No. This code is owned by the Tools team.